wire-ios-data-model/Source/Public/ZMMessage.h

```diff
@@ -67,8 +67,8 @@ typedef NS_CLOSED_ENUM(int16_t, ZMSystemMessageType) {
     ZMSystemMessageTypeFailedToAddParticipants,
     ZMSystemMessageTypeParticipantsRemoved,
     ZMSystemMessageTypeConversationNameChanged,
-    ZMSystemMessageTypeConnectionRequest,
-    ZMSystemMessageTypeConnectionUpdate,
+    ZMSystemMessageTypeConnectionRequest __deprecated_enum_msg("deprecated"),
+    ZMSystemMessageTypeConnectionUpdate __deprecated_enum_msg("deprecated"),
```

Comment:
I took this cases as deprecate because of `ConversationSystemMessageCell`, that says the cases are deprecated.

Reply:
as far as I understand we can't remove them, right?, so maybe add more info on the deprecated message to keep track of why we can't remove them

Reply:
I have added a longer comment on top. For the two cases I just added the compiler attribute, but they were deprecated before (see `ConversationSystemMessageCell`), so I don't know the exact details-